Kochbuch/src/app/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="userProfile">
+    <div class="col-md-6 well well-sm">
+        <h3>
+            Gibonni_123
+        </h3>
+        <img class="img-rounded" src="assets/user.png" />
+        <div>
+            <label>
+                    Benutzer seit:
+                </label>
+            <p>
+                15. Februar 2017
+            </p>
+            <label>
+                    Beschreibung:
+                </label>
+            <p>
+                Mein Name ist Gibonni und ich komme aus der wunderschönen gegend neben Bologna, Italien. Zu meinen vorlieben gehören die traditionelen italienischen gerichte wie diverse Pasta und Bruschetta Sorten, aber auch ven den typischen Italienischen Salate schweife
+                ich nicht ab.
+            </p>
+        </div>
+    </div>
+    <div style="min-height: 50em; background: gray;" class="col-md-6">
```

Comment:
Inline-Styles bitte vermeiden, entweder eine geeignete Bootstrap-Klasse suchen oder in die Style Datei auslagern